src/index.html

```diff
@@ -21,7 +24,372 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
```

Comment:
Following the BEM methodology, your body is a block, not an element, so it's better to use only `page` as the class, instead of `page__body`